shared/models/address/address_us.yml

```diff
@@ -118,6 +104,20 @@ properties:
   date_modified:
     $ref: "../../../shared/attributes/timestamps.yml#/date_modified"
 
+  recipient_moved:
```

Comment:
lol any particular reason you moved this?

Reply:
yeah it was in the wrong spot!